components/match2/wikis/criticalops/match_group_input_custom.lua

```diff
@@ -89,7 +89,7 @@ function CustomMatchGroupInput.processOpponent(record, date)
 		or Opponent.blank()
 
 	-- Convert byes to literals
-	if opponent.type == Opponent.team and opponent.template:lower() == 'bye' then
+	if opponent.type == (Opponent.team or Opponent.individual) and opponent.template:lower() == 'bye' then
```

Comment:
```suggestion
	if opponent.type == Opponent.team and opponent.template:lower() == 'bye' or opponent.name:lower() == 'bye' then
```

Reply:
what should I do, replacing with the new line creates **92: attempt to index field 'name' (a nil value).**
92 = this line

Reply:
```suggestion
	if opponent.type == Opponent.team and opponent.template:lower() == 'bye'
		or opponent.type == Opponent.solo and opponent.name:lower() == 'bye' then

```